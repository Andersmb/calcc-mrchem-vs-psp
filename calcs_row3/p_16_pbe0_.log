Starting job 5688128 on c2-[33,38-39],c3-6,c5-40 at Fri May 20 04:09:58 CEST 2022

Copying p_16_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying p_16_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688128      p_16_pbe0_        180                                             01:44:01      0:0 
5688128.bat+      batch         36        1   00:00:04          0   00:00:04   01:44:01      0:0 
5688128.ext+     extern        180        5   00:00:00          3   00:00:00   01:44:01      0:0 
5688128.0      mrchem.x        180       30   01:43:13         29   04:58:23   01:43:23      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688128                                                                          
5688128.bat+     19272K          0     19272K        0              0          0 
5688128.ext+          0          3          0        0              3          0 
5688128.0     35007504K         29 114609001+        0              9          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688128                                                                                               
5688128.bat+       19.42M               0         19.42M        0.08M                0          0.08M 
5688128.ext+        0.00M               3          0.00M            0                3              0 
5688128.0           6.75M               4          6.74M      274.77M                3        164.72M 

Job 5688128 completed at Fri May 20 05:54:00 CEST 2022
