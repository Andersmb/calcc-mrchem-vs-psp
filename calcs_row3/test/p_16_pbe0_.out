                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       N            :        0.000000        0.000000        2.927020
    1       N            :        0.000000        2.534878       -1.463380
    2       N            :        0.000000       -2.534878       -1.463380
    3       P            :        0.000000        2.807755        1.621031
    4       P            :        0.000000        0.000000       -3.242179
    5       P            :        0.000000       -2.807755        1.621031
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              33
 Beta electrons          :                                              33
 Total electrons         :                                              66
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999922033e-01
    1    1    a          :                              1.000001116417e+00
    2    1    a          :                              9.999999513492e-01
    3    1    a          :                              1.000000170434e+00
    4    1    a          :                              1.000000899373e+00
    5    1    a          :                              9.999998932246e-01
    6    1    a          :                              1.000000976847e+00
    7    1    a          :                              9.999990018319e-01
    8    1    a          :                              1.000000964755e+00
    9    1    a          :                              1.000000094306e+00
   10    1    a          :                              9.999995033151e-01
   11    1    a          :                              9.999999461161e-01
   12    1    a          :                              9.999991914479e-01
   13    1    a          :                              1.000000297593e+00
   14    1    a          :                              1.000000929420e+00
   15    1    a          :                              1.000000391254e+00
   16    1    a          :                              9.999996048072e-01
   17    1    a          :                              9.999998953949e-01
   18    1    a          :                              9.999994904207e-01
   19    1    a          :                              1.000000568423e+00
   20    1    a          :                              1.000000359571e+00
   21    1    a          :                              1.000000615472e+00
   22    1    a          :                              1.000001342793e+00
   23    1    a          :                              9.999998889657e-01
   24    1    a          :                              9.999990791565e-01
   25    1    a          :                              1.000000511969e+00
   26    1    a          :                              9.999999934694e-01
   27    1    a          :                              9.999992660738e-01
   28    1    a          :                              1.000000076366e+00
   29    1    a          :                              1.000000343173e+00
   30    1    a          :                              1.000000159207e+00
   31    1    a          :                              9.999999430091e-01
   32    1    a          :                              9.999992836534e-01
   33    1    b          :                              1.000000272136e+00
   34    1    b          :                              1.000000301105e+00
   35    1    b          :                              9.999999954560e-01
   36    1    b          :                              1.000000164599e+00
   37    1    b          :                              1.000000909839e+00
   38    1    b          :                              1.000000105080e+00
   39    1    b          :                              1.000000827537e+00
   40    1    b          :                              1.000000211467e+00
   41    1    b          :                              9.999994300477e-01
   42    1    b          :                              9.999995459667e-01
   43    1    b          :                              9.999994483169e-01
   44    1    b          :                              1.000001666448e+00
   45    1    b          :                              9.999988580162e-01
   46    1    b          :                              1.000000297873e+00
   47    1    b          :                              1.000000895762e+00
   48    1    b          :                              1.000000567330e+00
   49    1    b          :                              9.999996999938e-01
   50    1    b          :                              9.999999967457e-01
   51    1    b          :                              9.999995498625e-01
   52    1    b          :                              1.000000550328e+00
   53    1    b          :                              1.000000399999e+00
   54    1    b          :                              1.000000789302e+00
   55    1    b          :                              1.000001341807e+00
   56    1    b          :                              1.000001289505e+00
   57    1    b          :                              9.999997043292e-01
   58    1    b          :                              1.000000941092e+00
   59    1    b          :                              1.000000006829e+00
   60    1    b          :                              9.999992660738e-01
   61    1    b          :                              1.000000076366e+00
   62    1    b          :                              1.000000548158e+00
   63    1    b          :                              1.000000239283e+00
   64    1    b          :                              9.999999459646e-01
   65    1    b          :                              9.999992830257e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)               1176.550994039784
 E-N energy              :            (au)              -3610.902194153476
 Coulomb energy          :            (au)                939.504045138723
 Exchange energy         :            (au)                -21.969348597241
 X-C energy              :            (au)                -68.090227772232
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                403.707792351489
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1584.906731344442
 Nuclear energy          :            (au)                403.707792351489
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.181198938993e+03
                         :      (kcal/mol)             -7.412135189908e+05
                         :        (kJ/mol)             -3.101237365470e+06
                         :            (eV)             -3.214205914959e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -76.943363336286
    1    1    a          :            (au)                -76.924269970215
    2    1    a          :            (au)                -76.919946344089
    3    1    a          :            (au)                -13.975604002017
    4    1    a          :            (au)                -13.971677110329
    5    1    a          :            (au)                -13.969030977912
    6    1    a          :            (au)                 -6.760118708737
    7    1    a          :            (au)                 -6.758715001636
    8    1    a          :            (au)                 -6.753032125228
    9    1    a          :            (au)                 -4.910799406622
   10    1    a          :            (au)                 -4.907981821615
   11    1    a          :            (au)                 -4.904703525139
   12    1    a          :            (au)                 -4.902549517777
   13    1    a          :            (au)                 -4.902093480535
   14    1    a          :            (au)                 -4.901668768989
   15    1    a          :            (au)                 -4.898057298383
   16    1    a          :            (au)                 -4.896370874968
   17    1    a          :            (au)                 -4.893385723787
   18    1    a          :            (au)                 -0.922702271766
   19    1    a          :            (au)                 -0.811262255413
   20    1    a          :            (au)                 -0.809879011001
   21    1    a          :            (au)                 -0.603968694930
   22    1    a          :            (au)                 -0.603441588129
   23    1    a          :            (au)                 -0.424795538808
   24    1    a          :            (au)                 -0.423878240238
   25    1    a          :            (au)                 -0.352841111650
   26    1    a          :            (au)                 -0.305036816427
   27    1    a          :            (au)                 -0.302870123470
   28    1    a          :            (au)                 -0.241434134978
   29    1    a          :            (au)                 -0.233552210183
   30    1    a          :            (au)                 -0.232256259369
   31    1    a          :            (au)                 -0.115765381631
   32    1    a          :            (au)                 -0.114167402262
   33    1    b          :            (au)                -76.943299282816
   34    1    b          :            (au)                -76.924344009821
   35    1    b          :            (au)                -76.919924958650
   36    1    b          :            (au)                -13.975602632085
   37    1    b          :            (au)                -13.971680951947
   38    1    b          :            (au)                -13.969028177724
   39    1    b          :            (au)                 -6.760094523148
   40    1    b          :            (au)                 -6.758735736333
   41    1    b          :            (au)                 -6.753031017824
   42    1    b          :            (au)                 -4.910800891243
   43    1    b          :            (au)                 -4.907982780192
   44    1    b          :            (au)                 -4.904702239764
   45    1    b          :            (au)                 -4.902580969533
   46    1    b          :            (au)                 -4.902054255085
   47    1    b          :            (au)                 -4.901665432318
   48    1    b          :            (au)                 -4.898062005361
   49    1    b          :            (au)                 -4.896348860573
   50    1    b          :            (au)                 -4.893382506138
   51    1    b          :            (au)                 -0.922701768105
   52    1    b          :            (au)                 -0.811259958074
   53    1    b          :            (au)                 -0.809878868009
   54    1    b          :            (au)                 -0.603965805231
   55    1    b          :            (au)                 -0.603440747656
   56    1    b          :            (au)                 -0.424807799537
   57    1    b          :            (au)                 -0.423875783631
   58    1    b          :            (au)                 -0.352840090250
   59    1    b          :            (au)                 -0.305036667148
   60    1    b          :            (au)                 -0.302869269686
   61    1    b          :            (au)                 -0.241435380303
   62    1    b          :            (au)                 -0.233552035856
   63    1    b          :            (au)                 -0.232256325364
   64    1    b          :            (au)                 -0.115765582937
   65    1    b          :            (au)                 -0.114167186778
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -687.182393533642
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          8.124038e+00       -1181.198938992953         -1.181199e+03
    1          5.873441e-01       -1187.591929589679         -6.392991e+00
    2          2.263927e-01       -1187.841252608700         -2.493230e-01
    3          9.423902e-02       -1187.884911355115         -4.365875e-02
    4          5.380340e-02       -1187.897847579467         -1.293622e-02
    5          7.231048e-02       -1187.907198250316         -9.350671e-03
    6          1.817060e-02       -1187.908250640885         -1.052391e-03
    7          2.179051e-02       -1187.908866417609         -6.157767e-04
    8          9.929758e-03       -1187.908992506084         -1.260885e-04
    9          1.175560e-02       -1187.909088940421         -9.643434e-05
   10          5.235468e-03       -1187.909121182532         -3.224211e-05
   11          3.266216e-03       -1187.909135044336         -1.386180e-05
   12          1.164954e-03       -1187.909137863123         -2.818787e-06
   13          8.370261e-04       -1187.909138902395         -1.039272e-06
   14          7.232239e-04       -1187.909139549741         -6.473458e-07
   15          3.841682e-04       -1187.909139740770         -1.910291e-07
   16          3.245841e-04       -1187.909139843063         -1.022936e-07
   17          2.307797e-04       -1187.909139887455         -4.439175e-08
   18          1.658281e-04       -1187.909139913104         -2.564843e-08
---------------------------------------------------------------------------
                      SCF converged in 18 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       N            :        0.000000        0.000000        2.927020
    1       N            :        0.000000        2.534878       -1.463380
    2       N            :        0.000000       -2.534878       -1.463380
    3       P            :        0.000000        2.807755        1.621031
    4       P            :        0.000000        0.000000       -3.242179
    5       P            :        0.000000       -2.807755        1.621031
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)               1185.521386236548
 E-N energy              :            (au)              -3628.216707562151
 Coulomb energy          :            (au)                941.018919416913
 Exchange energy         :            (au)                -21.923754163663
 X-C energy              :            (au)                -68.016776192240
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                403.707792351489
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1591.616932264593
 Nuclear energy          :            (au)                403.707792351489
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.187909139913e+03
                         :      (kcal/mol)             -7.454242336071e+05
                         :        (kJ/mol)             -3.118854995436e+06
                         :            (eV)             -3.232465301059e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.286358373031
    1    1    a          :            (au)                -77.286316652654
    2    1    a          :            (au)                -77.286315118434
    3    1    a          :            (au)                -14.401302943888
    4    1    a          :            (au)                -14.401299055077
    5    1    a          :            (au)                -14.401285551667
    6    1    a          :            (au)                 -6.677811918976
    7    1    a          :            (au)                 -6.677774667752
    8    1    a          :            (au)                 -6.677772996676
    9    1    a          :            (au)                 -4.815852288053
   10    1    a          :            (au)                 -4.815815099505
   11    1    a          :            (au)                 -4.815813057389
   12    1    a          :            (au)                 -4.812926719382
   13    1    a          :            (au)                 -4.812889348208
   14    1    a          :            (au)                 -4.812887741264
   15    1    a          :            (au)                 -4.808665220100
   16    1    a          :            (au)                 -4.808628991900
   17    1    a          :            (au)                 -4.808620048566
   18    1    a          :            (au)                 -0.961136580966
   19    1    a          :            (au)                 -0.881239657827
   20    1    a          :            (au)                 -0.881215757899
   21    1    a          :            (au)                 -0.653203350789
   22    1    a          :            (au)                 -0.653171662792
   23    1    a          :            (au)                 -0.515891933563
   24    1    a          :            (au)                 -0.433770104402
   25    1    a          :            (au)                 -0.430858510710
   26    1    a          :            (au)                 -0.400461171113
   27    1    a          :            (au)                 -0.348036814531
   28    1    a          :            (au)                 -0.348017875956
   29    1    a          :            (au)                 -0.338349448641
   30    1    a          :            (au)                 -0.338345329759
   31    1    a          :            (au)                 -0.280139324492
   32    1    a          :            (au)                 -0.280119445882
   33    1    b          :            (au)                -77.286358371494
   34    1    b          :            (au)                -77.286316654220
   35    1    b          :            (au)                -77.286315119299
   36    1    b          :            (au)                -14.401302944125
   37    1    b          :            (au)                -14.401299058977
   38    1    b          :            (au)                -14.401285548803
   39    1    b          :            (au)                 -6.677811918404
   40    1    b          :            (au)                 -6.677774671589
   41    1    b          :            (au)                 -6.677772993455
   42    1    b          :            (au)                 -4.815852287856
   43    1    b          :            (au)                 -4.815815106877
   44    1    b          :            (au)                 -4.815813049571
   45    1    b          :            (au)                 -4.812926720718
   46    1    b          :            (au)                 -4.812889348230
   47    1    b          :            (au)                 -4.812887741396
   48    1    b          :            (au)                 -4.808665220646
   49    1    b          :            (au)                 -4.808628994357
   50    1    b          :            (au)                 -4.808620046023
   51    1    b          :            (au)                 -0.961136580822
   52    1    b          :            (au)                 -0.881239658259
   53    1    b          :            (au)                 -0.881215758247
   54    1    b          :            (au)                 -0.653203350475
   55    1    b          :            (au)                 -0.653171662010
   56    1    b          :            (au)                 -0.515891933839
   57    1    b          :            (au)                 -0.433770104593
   58    1    b          :            (au)                 -0.430858511026
   59    1    b          :            (au)                 -0.400461170082
   60    1    b          :            (au)                 -0.348036815543
   61    1    b          :            (au)                 -0.348017876966
   62    1    b          :            (au)                 -0.338349448977
   63    1    b          :            (au)                 -0.338345329093
   64    1    b          :            (au)                 -0.280139324378
   65    1    b          :            (au)                 -0.280119444467
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -692.304585526662
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000280       -0.000016       -0.000574
 Magnitude               :            (au)                        0.000638
                         :         (Debye)                        0.001623
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.000056
 Magnitude               :            (au)                        0.000056
                         :         (Debye)                        0.000143
---------------------------------------------------------------------------
 Total vector            :        0.000280       -0.000016       -0.000517
 Magnitude               :            (au)                        0.000588
                         :         (Debye)                        0.001495
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  1h 43m 18s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
